### ISSUE TITLE
Porkcord Icon Being Used for Roles

### ISSUE THREAD

**Issue by DracoClaw** [Snapperito/Settings-Icons#19]
The Powercord icon is being used next to role names in the new role menu when looking at the role perms. 

![image](https://user-images.githubusercontent.com/34274162/108634841-51188a80-7484-11eb-808f-5fbef6f3ce5f.png)

![image](https://user-images.githubusercontent.com/34274162/108634845-55dd3e80-7484-11eb-9dca-bd8be7245a16.png)

**Comment by SomeAspy** [comment]
I can probably figure out how to fix that. I'll add it to the pull request when I add my icons

**Comment by Snapperito** [comment]
> I can probably figure out how to fix that. I'll add it to the pull request when I add my icons

Me and draco already talked it out, and icon submissions are being accepted atm because of v2. 
You can make a pr with your icon and it will be included into v2 straight away though.

**Comment by Snapperito** [comment]
It seems like discord finally pushed this to stable. I'll look into a fix later today.

**Comment by leeprky** [comment]
```css
.container-23jqbX .list-I3aHXn .side-8zPYf6 [class="row-3GRgeK item-PXvHYJ themed-OHr7kt"]::before, [class="row-3GRgeK item-PXvHYJ selected-3s45Ha themed-OHr7kt"]::before {
    display: none;
 }
```

I Fixed it with that, prety sure it doesnt remove any other icons

**Comment by Snapperito** [comment]
reviving this project. will be fixed in 2.0
